tests/Scenarios/BorrowLimitScenTest.js

```diff
@@ -0,0 +1,3 @@
+const scenario = require('../Scenario');
```

Comment:
This file should have been ignored. It may have been included due to a case-sensitive file system? In either case, can you remove? It's auto-generated on test runs.

Reply:
Ah, yes I did have this in my gitignore but didn't realize it'd be auto generated. Will remove now.